dist/profile/manifests/buildmaster.pp

```diff
@@ -344,16 +344,16 @@
 # proxy_pass_match configurations
 ProxyRequests Off
 ProxyPreserveHost On
-ProxyPassMatch (.*)/api/json(/|$)(.*)  !
+ProxyPassMatch (.*)/api/(json|python)(/|$)(.*)  !
 ProxyPass / http://localhost:8080/ nocanon
 ProxyPassReverse / http://localhost:8080/
 ",
     aliases               => [
       {
-        # Send all api/json requests to `empty.json` to prevent abusive clients
+        # Send all api/json or api/python requests to `empty.json` to prevent abusive clients
         # (checkman) from receiving an invalid JSON response and repeatedly attempting
-        # to hammer us to get a better response
-        aliasmatch => '(.*)/api/json(/|$)(.*)',
+        # to hammer us to get a better response. Works for Python API as well.
+        aliasmatch => '(.*)/api/(json|python)(/|$)(.*)',
         path       => "${docroot}/empty.json",
```

Comment:
Well that will be invalid for Python.

Anyway we should not be redirecting to an empty response! Use an HTTP error code.

Reply:
It isn't! `{}` is valid JSON and, so far as I can tell, valid Python. =)

Reply:
The responses for api/json are specifically set because of abusive clients which don't respect any headers or other data. Do not attempt to apply logic here.

Reply:
And where are sources for this client? Have you filed a PR to fix it?